nextflow_schema.json

```
@@ -580,5 +472,14 @@
         {
             "$ref": "#/definitions/institutional_config_options"
         }
-    ]
-}
+    ],
+    "properties": {
+        "clusterOptions": {
+            "type": "string",
+            "description": "Arguments passed to Nextflow clusterOptions.",
+            "hidden": true,
+            "help_text": "UPPMAX profile only: Submit arbitrary SLURM options.",
```

```suggestion
            "help_text": "Add arbitrary options for job submission.",
```

I think we can remove the profile specific hint here. This works for every executor that has the `clusterOptions` config field,

---

@tmuylder 
Also, would you mind changing the all occurences of `clusterOptions` to `cluster_options`?
We changed the parameter naming to `snake_case` but apparently missed this one.